src/rgw/rgw_sal_motr.cc

```diff
@@ -2489,10 +2489,6 @@ int MotrObject::copy_object_same_zone(RGWObjectCtx& obj_ctx,
   if (tagging_drctv) {
     if (strcasecmp(tagging_drctv, "COPY") == 0) {
       rc = read_op->get_attr(dpp, RGW_ATTR_TAGS, tags_bl, y);
-      if (rc < 0) {
-        ldpp_dout(dpp, LOG_ERROR) <<__func__ << ": ERROR: read op for object tags failed rc=" << rc << dendl;
```

Comment:
keep log at debug level stating no tags seen for this src object

Reply:
Addressed.